src/i2c.c

```diff
@@ -0,0 +1,44 @@
+#include "i2c.h"
+
+#include "hardware/i2c.h"
+
+void
+c_i2c_init(mrb_vm *vm, mrb_value *v, int argc)
+{
+	i2c_init(I2C_PORT, 400 * 1000);
+  gpio_set_function(16, GPIO_FUNC_I2C);

```

Comment:
@aikawaYO Your code uses both hard tab and soft tab like these lines.
Please be consistent on the soft tab of "2 spaces"

Reply:
I'm sorry, my check mistake